### PR TITLE
update gon config

### PR DESCRIPTION
**Pull request by jkennedyvz**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by crizzitello** [comment]
I would rebase this on #120.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -170,8 +170,7 @@ jobs:
             if [ "${{ env.signRelease }}" == "true" ]; then
                 brew tap mitchellh/gon
                 brew install mitchellh/gon/gon jq
-                echo "${{ secrets.GON_CONF }}" | base64 -D -i - > notarize.json
-                env ID="${{ env.name }}.dmg" cat gon.json |jq '.notarize[0].path = env.ID'
+                env ID="${{ env.name }}.dmg" echo "${{ secrets.GON_CONF }}" | base64 -D -i - |jq '.notarize[0].path = env.ID' > notarize.json
```
Comment (crizzitello) [inline .github/workflows/ci.yaml]: should there be a space here ? 
```
|jq '.notarize[0].path = env.ID'
```
```
| jq '.notarize[0].path = env.ID'
```

**Review by crizzitello — COMMENTED** [review]
consider squashing those commits

File: CMakeLists.txt [hunk 2]
```diff
@@ -185,6 +185,7 @@ if(APPLE)
     else()
         add_custom_command(
             TARGET ashirt POST_BUILD
+            message(STATUS "Signing ID: ${NOTARIZE_AS}")
```
Comment (crizzitello) [inline CMakeLists.txt]: This has to go outside the add_custom_command call (line 186)

File: src/CMakeLists.txt [hunk 3]
```diff
@@ -69,6 +69,9 @@ if(APPLE)
         message(STATUS "Sign Bundle As: ${NOTARIZE_AS}")
         add_custom_command(
             TARGET ashirt POST_BUILD
+            COMMAND mkdir ${CMAKE_CURRENT_BINARY_DIR}/ashirt.app/Contents/Frameworks
+            COMMAND cp /usr/local/Cellar/brotli/1.0.9/lib/libbrotlicommon.1.0.9.dylib ${CMAKE_CURRENT_BINARY_DIR}/ashirt.app/Contents/Frameworks/libbrotlicommon.1.dylib
+            COMMAND codesign -v --timestamp --sign ${NOTARIZE_AS} ${CMAKE_CURRENT_BINARY_DIR}/ashirt.app/Contents/Frameworks/libbrotlicommon.1.dylib
```
Comment (jrozner) [inline src/CMakeLists.txt:74]: Do we still want to call this here?

**Review by jrozner — APPROVED** [review]

**Pull request merged**